readme.md

```diff
@@ -0,0 +1,31 @@
+## BeeYou Games Project
+
+This reposository stores the source project for the BeeYou Games Project.
+
+## Run Locally
+
+1. Download Unity Hub 
+
+```
+https://public-cdn.cloud.unity3d.com/hub/prod/UnityHubSetup.dmg
+```
+
+2. Download Unity `2021.3.15f1`
+
+```
+[Install correct version on Unity Hub](unityhub://2021.3.15f1/e8e88683f834)
+```
+
+3. Install Git-LFS
+
+```
+https://git-lfs.com/
+```
+
+4. Install Visual Studio 2022
```

Comment:
Maybe change this to any preferred version of visual studio or any other code editor. Because it's not mandatory to use visual studio in this project.

Reply:
alright, I'll change it

Reply:
just changed it, made the links clickable with angle brackets as well